src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.cs

```diff
@@ -455,29 +455,24 @@ public void HasServerIssues_IssuesChanged_RaisesPropertyChanged()
             var store = new Mock<ITaintStore>();
             var testSubject = CreateTestSubject(store: store);
 
-            var callCount = 0;
-            PropertyChangedEventArgs suppliedArgs = null;
-            testSubject.PropertyChanged += (sender, args) => { callCount++; suppliedArgs = args; };
+            var eventHandler = new Mock<PropertyChangedEventHandler>();
```

Comment:
suppliedArgs was no longer suitable; it was capturing the first NotifyPropertyChanged, and now we are raising two (one for the new property and one for the old `HasServerIssues`).